google/cloud/iam/samples/iam_samples.cc

```diff
@@ -208,6 +208,10 @@ std::string CreateServiceAccountKey(std::vector<std::string> const& argv) {
             TYPE_GOOGLE_CREDENTIALS_FILE,
         google::iam::admin::v1::ServiceAccountKeyAlgorithm::KEY_ALG_RSA_2048);
     if (!response) throw std::runtime_error(response.status().message());
+    // The private_key_data field in the response contains the 
+    // service account key in JSON format.
+    // TODO(Developer): Save the key to a secure location.
```

Comment:
We do not leave `TODO(Developer)` marks in our examples.  Also note that these examples are executable as-is, the parameters are received from the command-line, so the developer may not see this comment.  I think instead we should do something like:

```cc
  std::cout << "ServiceAccountKey successfully created: "
                  << response->DebugString() << "\n";
                  <<  "Please save the key in a secure location, as they cannot be downloaded later\n";
```

Reply:
Thank you for the insight. Modified the comment.